Test/git-issues/git-issue-3719.dfy

```diff
@@ -0,0 +1,19 @@
+// RUN: %exits-with 4 %baredafny verify %args "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+// Function want to verify something about
+opaque function inc(x:nat) : nat { x + 1 }
+
+// Intermediatary to force lambda
+predicate CheckThis(p: ()->bool) { p() }
```

Comment:
Formatting:
* Use one space after `:`
* Use no space before `:`
* Use a space before and after `->`

Reply:
Right, I had just taken the provided test without reformatting.